Content.Shared/GameObjects/Components/Observer/SharedGhostComponent.cs

```diff
@@ -11,6 +14,27 @@ public class SharedGhostComponent : Component, IActionBlocker
         public override string Name => "Ghost";
         public override uint? NetID => ContentNetIDs.GHOST;
 
+        [DataField("canReturnToBody")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool CanReturnToBody { get; set; }
```

Comment:
This is out of scope ofc, but I wonder if we could replace this by checking for `VisitingMindComponent` instead...

Reply:
Maybe but that would limit other things from preventing return to body (or overriding it).